### PR TITLE
Update sys-configurations-transact-sql.md

### PR DESCRIPTION
**Pull request by Rob-S**
Minor cosmetic changes for consistency and clarity.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@Rob-S : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/system-catalog-views/sys-configurations-transact-sql.md [hunk 1]
```diff
@@ -59,10 +59,8 @@ If the value equals the change for the configuration option you made but the **v
 
 There are  configuration options where the value and value_in_use may not be the same and this is expected behavior. For example:
 
-"max server memory (MB)" - The default configured value of 0 will show up as value_in_use = 2147483647
-"min server memory (MB)" - The default configured value of 0 may show up as value_in_use = 8 (32bit) or 16 (64bit). 
-
-In some cases, the **value_in_use** will be 0. In this situation, the "true" **value_in_use** is 8(32bit) or 16(64bit)
+"max server memory (MB)" - The default configured value of 0 will show up as **value_in_use** = 2147483647<br>
```
Comment (markingmyname) [inline docs/relational-databases/system-catalog-views/sys-configurations-transact-sql.md:62]:
```suggestion
"max server memory (MB)" - The default configured value of 0 shows up as **value_in_use** = 2147483647<br>

```

File: docs/relational-databases/system-catalog-views/sys-configurations-transact-sql.md [hunk 2]
```diff
@@ -59,10 +59,8 @@ If the value equals the change for the configuration option you made but the **v
 
 There are  configuration options where the value and value_in_use may not be the same and this is expected behavior. For example:
 
-"max server memory (MB)" - The default configured value of 0 will show up as value_in_use = 2147483647
-"min server memory (MB)" - The default configured value of 0 may show up as value_in_use = 8 (32bit) or 16 (64bit). 
-
-In some cases, the **value_in_use** will be 0. In this situation, the "true" **value_in_use** is 8(32bit) or 16(64bit)
+"max server memory (MB)" - The default configured value of 0 will show up as **value_in_use** = 2147483647<br>
+"min server memory (MB)" - The default configured value of 0 may show up as **value_in_use** = 8 (32bit) or 16 (64bit). In some cases, the **value_in_use** will be 0. In this situation, the "true" **value_in_use** is 8 (32bit) or 16 (64bit).
```
Comment (markingmyname) [inline docs/relational-databases/system-catalog-views/sys-configurations-transact-sql.md:64]:
```suggestion
"min server memory (MB)" - The default configured value of 0 may show up as **value_in_use** = 8 (32bit) or 16 (64bit). In some cases, the **value_in_use** is 0. In this situation, the "true" **value_in_use** is 8 (32bit) or 16 (64bit).

```

Comment (markingmyname) [inline docs/relational-databases/system-catalog-views/sys-configurations-transact-sql.md:64]:
```suggestion
"min server memory (MB)" - The default configured value of 0 may show up as **value_in_use** = 8 (32bit) or 16 (64bit). In some cases, the **value_in_use** is 0. In this situation, the "true" **value_in_use** is 8 (32bit) or 16 (64bit).

```

**Review by markingmyname — CHANGES_REQUESTED** [review]
Approved

**Comment by PRMerger18** [comment]
@markingmyname : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by markingmyname** [comment]
#sign-off

**Pull request merged**